lib/matplotlib/cbook.py

```diff
@@ -1941,8 +1941,8 @@ def _bootstrap_median(data, N=5000):
         M = len(data)
         percentiles = [2.5, 97.5]
 
-        ii = np.random.randint(M, size=(N, M))
-        bsData = x[ii]
+        bs_index = np.random.randint(M, size=(N, M))
+        bsData = data[bs_index]
```

Comment:
@anntzer that looks like it would work too. Not sure that performance would be significantly impacted in either direction. So I'm going to punt of figuring out which is better until the refactor associated with MEP 28 like a mentioned in my comment to Nelle.